### ISSUE TITLE
Multiple files in same settings manager

### ISSUE THREAD

**Issue by ljacqu** [AuthMe/ConfigMe#116]
As a user, I may like to separate my configuration into multiple files but in the code it is most convenient to use the same settings manager (since properties can already be separated by SettingsHolder classes). It would be nice to have a solution for this, maybe in 2.0, which would also be a good opportunity to tackle #12 (breaking changes only once)
